modules/router-store/src/router_store_module.ts

```diff
@@ -283,7 +284,7 @@ export class StoreRouterConnectingModule {
   }
 
   private setUpStateRollbackEvents(): void {
-    this.router.events.subscribe(e => {
+    this.router.events.subscribe((e: RouterEvent) => {
```

Comment:
VS Code shows correct typing completions for:

``` typescript
this.router.events.subscribe( e => { } );
```

I don't think adding typing here has any real value then again there's no real harm either.

Reply:
yes, but CI compile was throw err. My VS Code complain about this too.

Reply:
I think this was because of using angular v5. The error has gone when I upgrade to ng v6